packages/python-ccrawl/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=python-ccrawl
+pkgname=('python-ccrawl')
+_pkgname=ccrawl
+pkgver=1.2
+pkgrel=1
+pkgdesc='Ccrawl uses clang to build a database related to various C/C++ data structures.'
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-forensic')
+url='https://github.com/bdcht/ccrawl'
+license=('GPL3')
+makedepends=('git' 'python-pip')
+source=("git+https://github.com/bdcht/$_pkgname.git")
+sha512sums=('SKIP')
+
+prepare() {
+  cp -a "$_pkgname"{,-2}
+}
+
+build() {
+  cd "$srcdir/$_pkgname"
+
+  python setup.py build
+}
+
+package_python-ccrawl() {
+  depends=('python' 'python-virtualenv' 'python-click' 'python-traitlets' \
```

Comment:
It seems that `python-virtualenv` is not required but python-pymongo or python-requests are missing https://github.com/bdcht/ccrawl/blob/release/requirements.txt

Also maybe python-python-rapidjson and python-pyparsing https://github.com/bdcht/ccrawl/blob/bbb434bce370c2e42044225a6a91ba7b15e8d58e/setup.py#L28-L35

Reply:
python-python-rapidjson pukes. just fyi.